### PR TITLE
docs: add OpenAPI spec for hub-web

### PR DESCRIPTION
**Pull request by gsgalloway**
## Change Summary

Hand-off ownership of OpenAPI spec that Standard Crypto has written for the hub REST API.

Helpful UI for viewing spec: https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/standard-crypto/farcaster-js/develop/packages/farcaster-js-hub-rest/src/openapi/spec.yaml

## Merge Checklist

_Choose all relevant options below by adding an `x` now or at any time before submitting for review_

- [x] PR title adheres to the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard
- [ ] PR has a [changeset](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#35-adding-changesets)
- [ ] PR has been tagged with a change label(s) (i.e. documentation, feature, bugfix, or chore)
- [x] PR includes [documentation](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#32-writing-docs) if necessary.
- [x] All [commits have been signed](https://github.com/farcasterxyz/hub-monorepo/blob/main/CONTRIBUTING.md#22-signing-commits)

---

## PR-Codex overview
This PR updates the documentation for the Farcaster Hub Node.js package and adds an OpenAPI spec. 

### Detailed summary
- Added documentation for HTTP API endpoints
- Added OpenAPI spec for Farcaster Hub REST API

> The following files were skipped due to too many changes: `packages/hub-nodejs/spec.yaml`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by gsgalloway** [comment]
CC @sanjayprabhu

File: packages/hub-web/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,1838 @@
+openapi: 3.0.1
```
Comment (sanjayprabhu) [inline packages/hub-web/spec.yaml:1]: Could you please move this to `packages/hub-nodejs`?

File: packages/hub-web/README.md [hunk 2]
```diff
@@ -3,22 +3,28 @@
...
 yarn add axios @farcaster/core
 yarn add -D @types/axios 
 ```
 
 ## Documentation
-The HTTP API endpoints are [documented here](https://www.thehubble.xyz/docs/httpapi/httpapi.html). 
+
+The HTTP API endpoints are [documented here](https://www.thehubble.xyz/docs/httpapi/httpapi.html).
+
+An OpenAPI spec is provided [here](./spec.yaml).
```
Comment (sanjayprabhu) [inline packages/hub-web/README.md:24]: Please update `packages/hub-nodejs` instead. `hub-web` will be deprecated.

**Review by sanjayprabhu — COMMENTED** [review]
Thanks, could you please move the files to hub-nodejs? Should be good to merge after that.

**Comment by gsgalloway** [comment]
Hey @sanjayprabhu! Moved the changes over to `hub-nodejs`. Added a corresponding PR to the docs repo to include a link to view the OpenAPI spec in redocly: https://github.com/farcasterxyz/docs/pull/131

**Review by sanjayprabhu — APPROVED** [review]
Thank you!

**Pull request merged**